libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -339,7 +339,7 @@ void YamlConfiguration::parseProvenanceReportingYaml(
     return;
   }
 
-  if (!reportNode || !(reportNode->IsSequence())) {
```

Comment:
@apiri it is not a sequence for provenance report like processors. 
Provenance Reporting:
  scheduling strategy: TIMER_DRIVEN <=====we do not have -
  scheduling period: 1 sec
  port: 10001
  host: localhost
  port uuid: 471deef6-2a6e-4a7d-912a-81cc17e3a204
  batch size: 100